src/content/docs/data-apis/manage-data/manage-data-coming-new-relic.mdx

```diff
@@ -265,9 +265,9 @@ Some of the usage data in this UI can vary depending on your account. This infor
 
 For how to set alerts that will notify you when you're reaching data ingest levels you don't want to cross, see [Query and alert on usage data](/docs/accounts/accounts-billing/new-relic-one-pricing-users/usage-queries-alerts). For example, you might set an alert on logs, which can accumulate quickly in an active system.
 
-## Adjust your data ingest [#adjust-ingest]
+## Reduce your data ingest [#adjust-ingest]
 
-Here are some ideas for managing your data:
+All of the tools you use to report data to New Relic, whether they're our tools or third-party tools, will have various configuration options for adjusting the data reporting. We'll give some of the more common methods for reducing data ingest here, but we recommend looking at the docs for the specific tools you're using to learn other options. 
```

Comment:
I'm not sure about mentioning `third-party tools` here, feels like we're not in a good position to send people to the right place so it might be better to just skip the mention